runtime/packages/core-modules/assembly/executive.ts

```diff
@@ -47,6 +57,94 @@ export namespace Executive{
     export function finalizeBlock(): Header {
         return System.finalize();
     }
+    /**
+     * creates inherents from internal modules
+     * @param data inherents
+     */
+    export function createExtrinsics(data: InherentData): u8[] {
+        const timestamp: Inherent = Timestamp.createInherent(data);
```

Comment:
Even though TImestamp only produces inherent, I would like for us to call the Aura module as-well just for the purpose of the logic